tools/x-docs/src/docs/guides/components/cookbook/composing-components.md

```diff
@@ -0,0 +1,29 @@
+# Composing components
+
+Some components have slots that allow consumers to add extra elements in certain places. These can be other components or strings of HTML. One example of this is the [Teaser component's "actions" slot](https://github.com/financial-times/x-dash/tree/master/components/x-teaser/src/CustomSlot.jsx), which is typically used for a topic follow button.
+
+## Using a component's custom slot
+
+In the component's documentation will be a list of its properties and what it expects. In the Teaser component's case, its property `customSlot` will be rendered in an `o-teaser__action` element. This can be a JSX element or an HTML string.
```

Comment:
Turns out the `__action` bit is actually specific to one app so I should re-name it 🤦‍♂️